doc/_static/mpl.css

```diff
@@ -256,6 +315,30 @@ table.property-table td {
     width: 210px;
 }
 
+.sidebar-announcement {
+  border-left: solid 1px var(--light-gray);
+  border-bottom: solid 1px var(--light-gray);
+  border-right: solid 1px var(--light-gray);
+  border-bottom-right-radius: var(--border-radius);
+  border-bottom-left-radius: var(--border-radius);
+  box-shadow: var(--shadow);
+  margin-bottom: 20px;
+  padding: 28px 20px 20px;
+  position: relative;
+}
+
+.sidebar-announcement::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  height: 8px;
+  width: 100%;
+  background-color: var(--blue);
```

Comment:
This doesn't seem to do anything, so I think we should remove it and then not have to worry about having conflicting `--blue` variables.